src/features/merch/components/merch.module.scss

```
@@ -0,0 +1,68 @@
+@import "../../../styles/_variables";
```

Stylelint (17.10.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)